otx/algorithms/detection/adapters/mmdet/task.py

```diff
@@ -461,21 +455,12 @@ def hook(module, inp, outp):
         return prediction_results, metric
 
     # pylint: disable=too-many-statements
-    def export(
+    def _export_model(
         self,
-        export_type: ExportType,
-        output_model: ModelEntity,
-        precision: ModelPrecision = ModelPrecision.FP32,
-        dump_features: bool = True,
+        precision: ModelPrecision,
+        dump_features: bool,
```

Comment:
It's just a question. Previous `precision` and `dump_features` have default value, but they're removed.
It doesn't matter?

Reply:
Since it is called from parent export function and the parent export function contains default value, therefore we can assume that _export_model function always get real value from parent